Gemfile

```diff
@@ -74,7 +73,7 @@ gem 'bootstrap-sass', '~> 3.3.5'
 gem 'blacklight', '~> 6.3'
 gem 'blacklight-gallery', '~> 0.3'
 gem 'blacklight-maps', '0.2.0'
-gem 'blacklight-spotlight', '~> 0.18'
+gem 'blacklight-spotlight', '~> 0.19'
```

Comment:
(and here's a handy convenience task for next time: https://github.com/sul-dlss/exhibits/blob/master/lib/tasks/spotlight.rake#L46)